lib/browser/lighthouse.js

```diff
@@ -131,18 +131,16 @@ function parseLhResult(results) {
      * @type {auditInfo}
      */
     const auditInfo = { description, details };
+    const snippets = [];
 
     if (score) {
       successObj[id] = auditInfo;
     } else if (scoreDisplayMode === 'notApplicable') {
       notApplicableObj[id] = auditInfo;
     } else if (scoreDisplayMode !== 'manual') {
-      const snippets = [];
```

Comment:
**nit (non-blocking):** Isn't `snippets` only used inside this else/if clause?  Why not leave the declaration in here?

Reply:
a bad refactoring from my side, fixed!